packages/client-core/src/world/reducers/scenes/actions.ts

```diff
@@ -1,53 +1,53 @@
-import { SCENES_FETCHED_SUCCESS, SCENES_FETCHED_ERROR, SCENES_RETRIEVED, SET_CURRENT_SCENE } from '../actions'
```

Comment:
Please delete this file instead of commenting out if it's not going to be used now